docs/usage.rst

```diff
@@ -204,6 +204,24 @@ The default timeout is 60 seconds.
     CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0005_file_index.json
     CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0005_file_index.txt
     ..
+**File sizes and checksums**
```

Comment:
docs/usage.rst:207:Explicit markup ends without a blank line; unexpected unindent.

Reply:
BTW I'm seeing interesting failures locally:

```
__________________________________________________________________ test_download_files_https_pycurl __________________________________________________________________

    def test_download_files_https_pycurl():
        """Test download_files() command with https protocol using pycurl."""
        pycurl = pytest.importorskip("pycurl")  # noqa: F841
        test_file = "3005/0d0714743f0204ed3c0144941e6ce248.configFile.py"
        if os.path.isfile(test_file):
            os.remove(test_file)
        test_download_files = CliRunner()
        test_result = test_download_files.invoke(
            download_files, ["--recid", 3005, "--protocol", "https"]
        )
>       assert test_result.exit_code == 0
E       assert 1 == 0
E        +  where 1 = <Result error(1, 'Protocol "root" not supported or disabled in libcurl')>.exit_code

tests/test_cli_download_files.py:99: AssertionError
_________________________________________________________________ test_download_files_https_requests _________________________________________________________________

mocker = <pytest_mock.plugin.MockerFixture object at 0x7f3c388bf790>

    def test_download_files_https_requests(mocker):
        """Test download_files() command with https protocol using requests."""
        mocker.patch("cernopendata_client.downloader.pycurl_available", False)
        test_file = "3005/0d0714743f0204ed3c0144941e6ce248.configFile.py"
        if os.path.isfile(test_file):
            os.remove(test_file)
        test_download_files = CliRunner()
        test_result = test_download_files.invoke(
            download_files, ["--recid", 3005, "--protocol", "https"]
        )
>       assert test_result.exit_code == 0
E       assert 1 == 0
E        +  where 1 = <Result InvalidSchema("No connection adapters were found for 'root://eospublic.cern.ch//eos/opendata/cms/configuration-files/2011/0d0714743f0204ed3c0144941e6ce248.configFile.py'")>.exit_code

tests/test_cli_download_files.py:117: AssertionError
```

Shouldn't `root` be gone, and why are invoking XRootD in `test_download_files_https_requests()`?

Reply:
I fixed it, I had it in mind to replace the `root` with `https` in `get_files_list` after the `https` PR get merged.

Reply:
Seeing this one now:

```
________________________________________________________________________ test_validate_server ________________________________________________________________________

    def test_validate_server():
        """Test validate_server()."""
        assert validate_server("http://opendata.cern.ch") is True
        assert validate_server("http://opendata-dev.cern.ch") is True
        assert validate_server("http://0.0.0.0:5000") is True
>       pytest.raises(SystemExit, validate_server, "https://opendata.cern.ch")
E       Failed: DID NOT RAISE <class 'SystemExit'>

tests/test_validator.py:36: Failed
```

Does it pass for you locally?

Reply:
Yes 😅 , Have to fix the test for `https` as well.

Reply:
We can merge this one and address the rest in #92. It could be a bigger change, I left some comments there.